src/app/app.module.ts

```diff
@@ -26,11 +27,12 @@ import { PageNotFoundComponent } from './page-not-found/page-not-found.component
   declarations: [
     AppComponent,
     PageNotFoundComponent,
+    MenuBarComponent
```

Comment:
Let's rename MenuBarComponent to MenuComponent

Reply:
renamed